features/step_definitions/user_steps.rb

```diff
@@ -441,3 +441,15 @@
   @user.bio = ''
   @user.save
 end
+
+When(/^I look at user "(.*)"/) do |email|
+  @user = User.find_by_email email
+end
+
+Then(/^the tooltip (?:.*) (should not|should) be "(.*)"/) do |option, text|
```

Comment:
@tansaku - Could do yes - though for the negative case shouldn't the check still be the whole page?  I want to make sure that tooltip does not exist anywhere, whether it is the right place or not.

What would you limit it to?  `within ('#UsersList')`?  would that work or do I need to search to find #UsersList and then search within that?  
Also since it looks like I have to revisit anyway, I'd like to introduce https://github.com/kucaahbe/rspec-html-matchers and use have_tag (assuming it will work) since semantically that's what I'm looking for - a tag with a specific title.  I don't like the xpath expression I used because it doesn't express the intent well enough IMO